dao/src/test/java/com/carhouse/dao/impl/CarSaleDaoImplTest.java

```diff
@@ -72,8 +74,9 @@ void addCarSaleWithWrongReference() {
     void updateCarSale() {
         CarSale newCarSale = new CarSale(4, new BigDecimal(13200), new Date(), new User(1),
                 new Car(5));
-        carSaleDao.updateCarSale(newCarSale);
+        boolean isUpdated = carSaleDao.updateCarSale(newCarSale);
         CarSale obtainedCarSale = carSaleDao.getCarSale(4);
+        assertTrue(isUpdated);
```

Comment:
You're going to receive updated car BEFORE you checked that update has been successful.
Logically wrong

Reply:
I've fixed it by changing the return type of update method on void 
See the previous conversation

Reply:
It's wrong

Reply:
I've fixed it